### PR TITLE
Issue #SC-2086 fix: updated dial endpoint

### PR DESCRIPTION
**Pull request by krgauraw**

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_knowledge-mw-service.env [hunk 1]
```diff
@@ -21,7 +21,7 @@ sunbird_content_repo_api_base_url={{sunbird_content_repo_api_base_url}}
 sunbird_content_repo_api_key={{sunbird_content_repo_api_key}}
 sunbird_search_service_api_base_url={{knowledge_mw_sunbird_search_service_api_base_url | default('http://search-service:9000')}}
 sunbird_search_service_api_key={{sunbird_search_service_api_key}}
-sunbird_dial_repo_api_base_url="http://dial-service.{{namespace}}.svc.cluster.local:9000"
+sunbird_dial_repo_api_base_url={{sunbird_dial_repo_api_base_url}}
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/templates/sunbird_knowledge-mw-service.env:24]: Please change this to 
`sunbird_dial_repo_api_base_url=http://dial-service.{{namespace}}.svc.cluster.local:9000`

File: ansible/inventory/env/group_vars/all.yml [hunk 2]
```diff
@@ -192,7 +192,7 @@ keycloak_password: "{{core_vault_keycloak_password}}"
 sunbird_content_repo_api_base_url: "http://{{learningservice_ip}}:8080/learning-service"
 sunbird_content_repo_api_key: "{{ core_vault_sunbird_ekstep_api_key }}"
 sunbird_search_service_api_key: "{{ core_vault_sunbird_ekstep_api_key }}"
-sunbird_dial_repo_api_base_url: "{{sunbird_ekstep_api_base_url}}"
+sunbird_dial_repo_api_base_url: "http://dial-service.{{namespace}}.svc.cluster.local:9000"
```
Comment (keshavprasadms) [inline ansible/inventory/env/group_vars/all.yml:195]: Please remove this.

**Review by keshavprasadms — CHANGES_REQUESTED** [review]
Please see my comments.

File: ansible/inventory/env/group_vars/all.yml (hunk 2 above)
Comment (krgauraw) [reply ansible/inventory/env/group_vars/all.yml]: done

File: ansible/roles/stack-sunbird/templates/sunbird_knowledge-mw-service.env (hunk 1 above)
Comment (krgauraw) [reply ansible/roles/stack-sunbird/templates/sunbird_knowledge-mw-service.env]: done

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**